### PR TITLE
Add description for python script working directory input

### PR DESCRIPTION
**Pull request by joshmgross**

### CODE REVIEW

File: Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "loc.input.label.pythonInterpreter": "Python interpreter",
   "loc.input.help.pythonInterpreter": "Absolute path to the Python interpreter to use. If not specified, the task will use the interpreter in PATH.<br /> Run the [Use Python Version](https://go.microsoft.com/fwlink/?linkid=871498) task to add a version of Python to PATH.",
   "loc.input.label.workingDirectory": "Working directory",
+  "loc.input.help.workingDirectory": "The working directory where the script will run. When empty, the root of the repository (for builds) or artifacts (for releases) is used, which is the value of '$(System.DefaultWorkingDirectory)'.",
```
Comment (brcrista) [inline Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson]: I would word the second sentence like:

"If not specified, the value of `System.DefaultWorkingDirectory` will be used.  For builds, this variable defaults to the root of the repository.  For releases, it defaults to the root of the artifacts directory."

Comment (joshmgross) [reply Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson]: I'm fine with that language.

Updated wording to read:
`The working directory where the script will run. If not specified, the value of '$(System.DefaultWorkingDirectory)' will be used. For builds, this variable defaults to the root of the repository. For releases, it defaults to the root of the artifacts directory.`

File: Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "loc.input.label.pythonInterpreter": "Python interpreter",
   "loc.input.help.pythonInterpreter": "Absolute path to the Python interpreter to use. If not specified, the task will use the interpreter in PATH.<br /> Run the [Use Python Version](https://go.microsoft.com/fwlink/?linkid=871498) task to add a version of Python to PATH.",
   "loc.input.label.workingDirectory": "Working directory",
+  "loc.input.help.workingDirectory": "The working directory where the script will run. If not specified, the value of '$(System.DefaultWorkingDirectory)' will be used. For builds, this variable defaults to the root of the repository. For releases, it defaults to the root of the artifacts directory.",
```
Comment (hashtagchris) [inline Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson]: Use backticks (``) to get the `code` formatting Brian used.

File: Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (lkillgore) [reply Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson]: I'm not sure that $(System.DefaultWorkingDirectory) needs have the single quotes.  I think there's another reference in this file w/o them.  I'd be fine w/ either, but it should probably be consistent w/in the same file.

**Comment by hashtagchris** [comment]
Can you include a screenshot showing what the helptext looks like now?

File: Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (brcrista) [reply Tasks/PythonScriptV0/Strings/resources.resjson/en-US/resources.resjson]: Semantic nit, `$(System.DefaultWorkingDirectory)` _is_ the value of the variable `System.DefaultWorkingDirectory`.

**Comment by brcrista** [comment]
One more small comment, but looks good to me.  Thanks!

**Pull request merged**